src/Native/pkg/dir.props

```diff
@@ -22,4 +31,13 @@
     <Ubuntu1604NativePath Condition="'$(Ubuntu1604NativePath)' == ''">$(BuildNativePath)</Ubuntu1604NativePath>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
   </PropertyGroup>
+
+  <!-- Add path globs specific to native binaries to exclude unnecessary files in packages. -->
+  <ItemGroup>
+    <AdditionalLibPackageExcludes Include="%2A%2A\%2A.dbg" Condition="'$(OSGroupPath)'=='Linux'" />
```

Comment:
Good idea, and I think I'll do similar for the other globs. (I'll also move the condition to the `ItemGroup` to continue not adding these on Windows.)